aws-blog-event-driven-batch-analytics/README.md

```diff
@@ -0,0 +1,137 @@
+# Building Event Driven Batch Analytics on AWS
+
+## Overview
+
+This repository contains the code that supports the [AWS Big Data Blog Post](https://blogs.aws.amazon.com/bigdata/)
+
+### Usecase Description
+Yummy Foods has franchise stores all over the country. These franchise stores run on hetrogenous platforms and they submit cumulative transaction files to Yummy Foods at varuious cadence levels through out the day in tab delimited .tdf format. Some of these franchise stores due to their system limitations ocassionally send additional data starting with characters such as “----“.
+
+The requirement is to be able to update insights on the sales made by each franchise for a given item through out the day as soon as the complete list of franchise files from a given province are available. The number of franchises per province is fixed and seldom changes.
```

Comment:
Suggest "Yummy Foods needs to be able to update internal users on  sales made by each franchise for a given item throughout the day, and as soon as the complete list of franchise files from a given province are available. The number of franchises per province is fixed and seldom changes"